polynote-kernel/src/main/scala/polynote/kernel/PolyKernel.scala

```diff
@@ -173,6 +176,16 @@ class PolyKernel private[kernel] (
                       IO.pure(result)
                   }
                 }
+
+                for {
+                  _ <- symbolTable.drain()
+                  start <- clock.monotonic(MILLISECONDS)
```

Comment:
This is neat! Didn't even know this existed!

Reply:
Yeah it's pretty cool :) 
(once I figured out how to make one lol - `Clock.create` is not mentioned at all in the docs!).